pages/_app.tsx

```diff
@@ -4,13 +4,24 @@ import { wrapper } from '../redux/store';
 import { AppProps } from 'next/app';
 import Nav from '../components/Nav';
 
+import { setCookie, parseCookies } from 'nookies';
+
 const OpticTracker: FC<AppProps> = ({ Component, pageProps }): JSX.Element => {
-  const [useLight, setUseLight] = useState<boolean>(false);
+  const { useLight = 'dark' } = parseCookies();
```

Comment:
(nit) maybe `useLight` isn't so representative of the use now. Maybe `mode` or:

```suggestion
  const { appearanceMode = 'dark' } = parseCookies();
```

Reply:
I'm actually going to pull this out into a seperate PR, it's going to deal with state rehydration etc so i'll save it for another PR